app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    arguments = command.split()
+    if (command.startswith("cp ")
```

Comment:
you should check that command exactly is `cp` not just start with this letters

Reply:
Unpack the source and destination file paths for clarity

Reply:
Does unpack mean to create the variables?